yazi-config/preset/theme-dark.toml

```
@@ -76,8 +76,8 @@ unset_alt  = { fg = "red", bg = "gray" }
 # : Status bar {{{
 
 [status]
-separator_open  = ""
-separator_close = ""
+separator_left  = [ "", "" ]
```

Please use a dict instead to make it more readable:

```suggestion
separator_left  = { open = "", close = "" }
```